contracts/MemberTeller.sol

```diff
@@ -0,0 +1,68 @@
+pragma solidity ^0.8.7;
+import "./interfaces/IMemberToken.sol";
+
+contract MemberTeller {
+    IMemberToken public immutable memberToken;
+
+    bytes4 public constant ENCODED_SIG_ADD_OWNER =
+        bytes4(keccak256("addOwnerWithThreshold(address,uint256)"));
+    bytes4 public constant ENCODED_SIG_REMOVE_OWNER =
+        bytes4(keccak256("removeOwner(address,address,uint256)"));
+    bytes4 public constant ENCODED_SIG_SWAP_OWNER =
+        bytes4(keccak256("swapOwner(address,address,address)"));
+
+    uint8 internal constant SYNC_EVENT = 0x02;
+
+    constructor(address _memberToken) {
+        memberToken = IMemberToken(_memberToken);
+    }
+
+    function getSyncData() internal pure returns (bytes memory) {
```

Comment:
What is sync data?

Reply:
it creates the sync flag so Controller knows to skip side effects

Reply:
why is it called the sync flag? did you make that up or is that from something else?